src/java/org/apache/cassandra/db/compaction/UnifiedCompactionStrategy.java

```diff
@@ -403,20 +405,20 @@ public Set<CompactionSSTable> getFullyExpiredSSTables(int gcBefore)
     /// Used by CNDB where compaction aggregates come from etcd rather than the strategy.
     /// @return collection of `AbstractCompactionTask`, which could be either a `CompactionTask` or a `UnifiedCompactionTask`
     public synchronized Collection<AbstractCompactionTask> getNextBackgroundTasks(Collection<CompactionAggregate> aggregates, int gcBefore,
-                                                                                  CompactionObserver compositeCompactionObserver)
+                                                                                  @Nullable CompactionObserver additionObserver)
```

Comment:
Nit: `additionObserver` should change to `additionalObserver` here and in a few other places.